src/SFML/Window/OSX/HIDInputManager.hpp

```diff
@@ -145,40 +184,56 @@ class HIDInputManager : NonCopyable
     /// \brief Load the given keyboard into m_keys
     ///
     /// If the given keyboard has no key this function simply
-    /// returns. freeUp is _not_ called because this is not fatal.
+    /// returns without calling freeUp because this is not fatal.
     ///
     /// \param keyboard Keyboard to load
     ///
+    /// \see initializeKeyboard
+    ///
     ////////////////////////////////////////////////////////////
     void loadKeyboard(IOHIDDeviceRef keyboard);
 
     ////////////////////////////////////////////////////////////
     /// \brief Load the given key into m_keys
     ///
-    /// freeUp is _not_ called by this function.
+    /// On error, freeUp is _not_ called by this function.
     ///
     /// \param key Key to load
     ///
+    /// \see initializeKeyboard
+    ///
     ////////////////////////////////////////////////////////////
     void loadKey(IOHIDElementRef key);
 
+    ////////////////////////////////////////////////////////////
+    /// Regenerate the mappings from/to Key and Scancode.
+    ///
+    /// It is public to allow regular callback to forward the
+    /// information to the manager.
```

Comment:
"callbacks"?
also, which information?

Reply:
It updates the shared mapping and can be called (back) from outside. The phrasing isn't the best, so I'll adjust it.